examples/bri-3/src/bri/zeroKnowledgeProof/models/witness.ts

```diff
@@ -0,0 +1,7 @@
+export class Witness {
+  proof: any;
```

Comment:
does it have to be any? can it be string?

Reply:
In snarkjs, Proof is a json object with this format
 
 proof:  {
      pi_a: [
        '287015515026886850367000370304282418506884049051461050087416899760168881497',
        '18148864089385570498905488155229676521878972897521971323742422980058890911751',
        '1'
      ],
      pi_b: [
        [
          '4105642086103088183689535058970939306131735480592400919288550096864124823539',
          '9073470526838081637126023810680740679551786524097096458194988998006765175875'
        ],
        [
          '478070075091370606229554665611856500549347842347824569455531309060744385612',
          '17618482729486526802793459583506757327991535735572167103342901581209927978239'
        ],
        [ '1', '0' ]
      ],
      pi_c: [
        '9844961605053025613073899766400828870655794410013247320323691464876851190592',
        '14679854688986179956949270074292341730632138546772235800957114543397279031062',
        '1'
      ],
      protocol: 'groth16',
      curve: 'bn128'
    }

    Gnark also returns the pi_a, pi_b, pi_c properties but with different naming conventions.